blueocean-dashboard/src/main/js/components/RunDetailsHeader.jsx

```diff
@@ -76,14 +78,7 @@ class RunDetailsHeader extends Component {
                             </span>
                         </div>
                         : null }
-                        <div>
-                       { authors.length > 0 ?
-                                   <a className="authors" onClick={() => this.handleAuthorsClick()}>
-                                        Changes by {authors.map(
-                                        author => ` ${author}`)}
-                                   </a>
-                       : 'No changes' }
-                        </div>
+                        <ChangeSet {...changeProps} />
```

Comment:
I do not understand what you are saying.

I only extracted a component to be standalone. The properties are only passed through not not created in a different location. Actually by extracting it it makes it much more maintainable then before.

If you refer to ...changeProps that is personal style and we have no written guidelines.